src/etc/inc/notices.inc

```diff
@@ -410,10 +410,64 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 	log_error(sprintf(gettext("Message sent to %s OK"), $to));
 	return;
 }
+/****f* notices/notify_via_telegram
+ * NAME
+ *   notify_via_telegram
+ * INPUTS
+ *	 notification string to send to Telegram via API
+ * RESULT
+ *   returns NULL if message was sent
+ ******/
+
+function notify_via_telegram($message, $force = false){
+	global $config;
+	if (isset($config['notifications']['telegram']['disable']) && !$force) {
+		return;
+	}
+	if (!$config['notifications']['telegram']['api']) {
+		return;
+	}
+	if (!$config['notifications']['telegram']['chatid']) {
+		return;
+	}
```

Comment:
All of these could be combined into one test rather than using three separate tests. For example: if ((a && !b) || !c || !d)

Just be sure to use the correct parenthesis, line breaks, indenting, and so on when making the changes.